packages/account/src/providers/provider.ts

```diff
@@ -86,24 +93,27 @@ export type ContractResult = {
   bytecode: string;
 };
 
-type ConsensusParameters = {
-  contractMaxSize: BN;
-  maxInputs: BN;
-  maxOutputs: BN;
-  maxWitnesses: BN;
-  maxGasPerTx: BN;
-  maxScriptLength: BN;
-  maxScriptDataLength: BN;
-  maxStorageSlots: BN;
-  maxPredicateLength: BN;
-  maxPredicateDataLength: BN;
-  maxGasPerPredicate: BN;
-  gasPriceFactor: BN;
-  gasPerByte: BN;
-  maxMessageDataLength: BN;
+type ModifyStringToBN<T> = {
+  [P in keyof T]: P extends 'version' ? T[P] : T[P] extends string ? BN : T[P];
+};
+
+export type FeeParameters = Omit<GqlFeeParameters, '__typename'>;
```

Comment:
If you modify the `codegen.json` file to include the below, you wouldn't need these intermediary types that get rid of `__typename`:
```ts
{
      ...
      "plugins": [
        {
          "typescript": {
            "skipTypename": true
          }
        },
        ...
      ],
    ...
}
```

Reply:
@nedsalk Nice suggestion. Let me try that.

Reply:
@nedsalk There is a place where `__typename` is used:

- https://github.com/FuelLabs/fuels-ts/blob/master/packages/account/src/providers/utils/gas.ts#L27-L31

Reply:
Hmm, I believe that should be `type` and not `__typename`.

Reply:
This can be done on a follow-up PR if necessary. 👍 

cc @Torres-ssf